packages/openapi-v3/src/__tests__/unit/decorators/request-body/request-body.decorator.unit.ts

```diff
@@ -16,7 +16,7 @@ describe('requestBody decorator', () => {
       };
       class MyController {
         @post('/greeting')
-        greet(@requestBody(requestSpec) name: string) {}
+        greet(@requestBody2(requestSpec) name: string) { }
```

Comment:
It would be better to use `@requestBody` for all flavors.

Reply:
@raymondfeng see the section "Naming" in the spike.md file, I started a discussion there and proposed staying with `@requestBody()` since it doesn't break the existing usage.